### ISSUE TITLE
for your information: world without trees

### ISSUE THREAD

**Issue by jokish63** [Kashdeya/Knob-Control#2]
I've loaded my standard world, and I could not see any tree although I should direct standing next to a birch forest.
So I have explored that world and suprisingly I could not found any tree in the whole world.
I have explored a 10000 by 10000 area - no tree at all.
Installed is the recommended forge 2011 and the latest version from your mod in MC 1.10.2

**Comment by Kashdeya** [comment]
change the config file for trees. the update that is out has a few config options that are backwards. so if the disable trees config is set to false change it to true.

the next update is fixing all the wording in the config.

**Comment by jokish63** [comment]
I would like to make a suggestion. It not just with your mod. I think it it is a problem in general. 
For example: Functions and effects. In MC the Particles could turned off in general but I like them, just the Particle in the field of view bothers me a lot. If I found a mod which turns that off I and other mod which I use has also a particle adjustment  so I turn it off but I don't know exactly, is there an effect to the new mods or MC when I set it to false.
So my suggestion is it possible to summarize some things under a function. So I can switch off the hole function completely if I do not need them in genral and nothing do happen to other mods or MC, or I turn off just individual subfunctions or effects and must be aware sideeffects. I hope I could  explain wht I mean.
